Games/core/Message.js

```diff
@@ -83,7 +83,7 @@ module.exports = class Message {
     if (!version) version = this;
 
     if (version.isServer) senderId = "server";
-    else if (version.anonymous) senderId = "anonymous";
+    else if (version.anonymous && (player.alive || (!player.alive && version.sender.alive))) senderId = "anonymous";
```

Comment:
dead can communicate with each other non-anon here, is it intended?

e.g. LK day, dead talks, someone gets revived next day is this ok